aspnetcore/mvc/advanced/custom-model-binding/samples/3.x/CustomModelBindingSample/Binders/AuthorEntityBinder.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Threading.Tasks;
+using CustomModelBindingSample.Data;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace CustomModelBindingSample.Binders
+{
+    #region snippet_Class
+    public class AuthorEntityBinder : IModelBinder
+    {
+        private readonly AuthorContext _context;
```

Comment:
All existing uses of `db` have been replaced with `context` to align with the other samples that I've seen.